planet-express/java/src/Crew.java

```diff
@@ -0,0 +1,74 @@
+
+public class Crew {
+	int thirst = 50;
+	int work = 50;
+	int horde = 50;
+	int hunger = 50;
+	int receipts = 50;
+	boolean stable = true;
+	int score = 50;
+	
+	public void drink() {
+		thirst -= 5;
+		work += 5;
+	}
+
+	public void deliver() {
```

Comment:
Hmm i'm curious if the indentation that is showing here is how it looks in your IDE?